spec/rubocop/cop/lint/non_atomic_file_operation_spec.rb

```diff
@@ -252,6 +252,56 @@
     RUBY
   end
 
+  context 'with fully-qualified constant names' do
+    it 'registers an offense when existence check uses fully-quailified constant name' do
```

Comment:
This seems like a typo.

```suggestion
    it 'registers an offense when existence check uses fully qualified constant name' do
```